spec/lex.dd

```diff
@@ -340,7 +340,37 @@ $(GNAME StringPostfix):
     $(B d)
 
 $(GNAME DelimitedString):
-    $(B q") $(I Delimiter) $(GLINK WysiwygCharacters)$(OPT) $(I MatchingDelimiter) $(B ")
+    $(B q") $(GLINK Delimiter) $(GLINK WysiwygCharacters)$(OPT) $(GLINK MatchingDelimiter) $(B ")
```

Comment:
There should be a newline before and after `$(GLINK WysiwygCharacters)$(OPT)`, though that might be out of the scope of this PR.